tests/SHA-VERIFICATION-REPORT.md

```
@@ -0,0 +1,126 @@
+- [x] ok 3 rejects SHA mismatch (one bit flipped)
+- [x] ok 4 rejects @v* tag references
+- [x] ok 5 rejects @main branch references
+- [x] ok 6 rejects pending-manifest entries that are referenced
+- [x] ok 7 rejects deprecated actions (Appendix B.2)
+- [x] ok 8 rejects unknown actions not present in either manifest
+- [x] ok 9 allows local references (./actions/foo)
+- [x] ok 10 scans actions/ subtree, not only workflows/
+- [x] ok 11 rejects manifest.yml that itself contains placeholder SHA
+
+### verify-sha-consistency.sh (live run)
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)